caddytest/caddytest.go

```diff
@@ -369,5 +375,53 @@ func AssertAdapt(t *testing.T, rawConfig string, adapterName string, expectedRes
 			}
 		}
 		t.Fail()
+
+		fmt.Println(orderedResponse)
 	}
 }
+
+type load struct {
+	Apps apps `json:"apps"`
+}
+type apps struct {
+	HTTP hTTP        `json:"http"`
+	TLS  interface{} `json:"tls,omitempty"`
+}
+type hTTP struct {
```

Comment:
This casing is weird. I know why you did it like this, but maybe just rename it instead, like `httpServers` or something.

Reply:
Yeah it felt iffy to me too:)